source/linear-algebra/exercises/outcomes/MX/MX2/generator.sage

```diff
@@ -43,6 +43,7 @@ class Generator(BaseGenerator):
             "invertible": False,
             "label": labels[1],
             "vector_eq": TBIL.VectorEquation(m),
+            "ordinal": ordinal_string,
```

Comment:
This will fix the non-invertible question so it doesn't ask for a [blank] column of the inverse.